jit_log: Extract a function that prefixes all lines of a string with another string.

ZolotukhinM opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* #24356 Module: add dump function that recursively prints contents of the module.
* **#24355 jit_log: Extract a function that prefixes all lines of a string with another string.**

Pull Request resolved: https://github.com/pytorch/pytorch/pull/24355

Differential Revision: [D16864134](https://our.internmc.facebook.com/intern/diff/D16864134)

Commit:
jit_log: Extract a function that prefixes all lines of a string with another string.

Krovatkin reviewed (approved):
:shipit: we could consider moving it to `StringUtils.h` later on, I dunno.

Commit:
Update on "jit_log: Extract a function that prefixes all lines of a string with another string."

ZolotukhinM commented:
> :shipit: we could consider moving it to `StringUtils.h` later on, I dunno.

I looked at `StringUtils.h`, but everything there seemed more generic than this function, so I hesitate to move it there. I wouldn't object if anyone wants to do it though:)

Commit:
Rename 'debug_print' to 'dump' on "jit_log: Extract a function that prefixes all lines of a string with another string."


Pull Request resolved: https://github.com/pytorch/pytorch/pull/24355

Differential Revision: [D16824512](https://our.internmc.facebook.com/intern/diff/D16824512)

Krovatkin commented:
> I looked at StringUtils.h, but everything there seemed more generic than this function, so I hesitate to move it there. I wouldn't object if anyone wants to do it though:)

Fine by me 😄
